rafthttp/peer.go

```diff
@@ -271,12 +264,14 @@ func (p *peer) pick(m raftpb.Message) (writec chan<- raftpb.Message, picked stri
 	// stream for a long time, only use one of the N pipelines to send MsgSnap.
 	if isMsgSnap(m) {
 		return p.pipeline.msgc, pipelineMsg
-	} else if writec, ok = p.msgAppWriter.writec(); ok && canUseMsgAppStream(m) {
```

Comment:
No. I mean before this change, the func will return `streamApp`. After this change, this will return `streamAppV2` instead. Do you mean that the previous picked name will be incorrect when the writec is actually v2?